synapse/lib/view.py

```diff
@@ -1000,7 +1000,10 @@ async def storm(self, text, opts=None):
         keepalive = opts.get('keepalive')
         if keepalive is not None and keepalive <= 0:
             raise s_exc.BadArg(mesg=f'keepalive must be > 0; got {keepalive}')
-        synt = await self.core.boss.promote('storm', user=user, info=taskinfo, taskiden=taskiden)
+
+        synt = None
+        if opts.get('promote', True):
```

Comment:
Interesting. This option technically allows any user to create invisible queries, no?

Reply:
removed